xdsl/tools/command_line_tool.py

```diff
@@ -442,6 +447,7 @@ def get_stencil_bufferize():
         "convert-scf-to-riscv-scf": get_convert_scf_to_riscv_scf,
         "convert-snitch-stream-to-snitch": get_convert_snitch_stream_to_snitch,
         "convert-stencil-to-csl-stencil": get_convert_stencil_to_csl_stencil,
+        "convert-stencil-to-csl": get_convert_stencil_to_csl,
```

Comment:
I would recommend calling it `test-convert-stencil-to-csl`, to minimise the confusion of alternative ways of getting to places. Ideally the csl work would actually have its own driver, and not use xdsl-opt, something like the `toy` command line tool that is toy-specific.

Reply:
Renamed as suggested. 

Though, this is less of a driver, and more just a way to keep track of the order of passes developed so far.

Reply:
> Though, this is less of a driver, and more just a way to keep track of the order of passes developed so far.

Well... Isn't that what the "driver" is though?

This would also allow us to add some simple logic to print the layout and program to multiple files and such. I guess we should discuss how to build actual compiler tools with xDSL (where you get an interface like `stencilcc --march=cs2 my_input_file.mlir --out_dir=out` that will write multiple files into `./out` for the csl stencil work)